tools/CouchDB_Import_Demographics.php

```diff
@@ -153,10 +143,9 @@ function _generateQuery() {
                                 LEFT JOIN participant_status_options pso ON (pso.ID=ps.participant_status)
                                 LEFT JOIN feedback_bvl_thread fbt ON (fbt.CandID=c.CandID) 
                                 LEFT JOIN feedback_bvl_entry fbe ON (fbe.FeedbackID=fbt.FeedbackID)";
-
         $groupBy=" GROUP BY s.ID, 
-                            c.DoB,
-							c.CandID, 
+                            c.DoB, 
+		            c.CandID, 
                             c.PSCID, 
```

Comment:
extra space at end of line 147
missing space at line 148

Reply:
@PapillonMcGill thanks for you review! will make these changes :)